scout/server/blueprints/cases/views.py

```diff
@@ -52,12 +53,13 @@
 @templated("cases/index.html")
 def index():
     """Display a list of all user institutes."""
-    institute_objs = user_institutes(store, current_user)
-    institutes_count = (
-        (institute_obj, sum(1 for i in store.cases(collaborator=institute_obj["_id"])))
-        for institute_obj in institute_objs
-        if institute_obj
-    )
+    institutes_count = []
+    for item in user_cases(store, current_user):
+        institute_obj = store.institute(item["_id"]["institute"])
+        if not institute_obj:
+            continue
+        institutes_count.append((institute_obj, item["count"]))
+
```

Comment:
We loose the institute sort now; I guess it was implicit before from looping over the institutes, but now they appear in user case order? Anyway, institute list needs a sort. `internal_id` is convenient enough for admins, and I guess regular users usually don't have that many, otherwise `display_name` sorting would be in order. Either is fine!